uzdevums2.py

```diff
@@ -0,0 +1,3 @@
+a = 5
+b = 9
+print(b%a)
```

Comment:
Starp mainīgajiem un darbībām liekam atstarpi:
```suggestion
print(b % a)
```